server/plugin.go

```diff
@@ -290,27 +289,18 @@ func (p *Plugin) postReplyIfNeeded(postID, message, todo string) {
 	}
 }
 
-func (p *Plugin) handleList(w http.ResponseWriter, r *http.Request) {
+func (p *Plugin) handleLists(w http.ResponseWriter, r *http.Request) {
 	userID := r.Header.Get("Mattermost-User-ID")
 
-	listInput := r.URL.Query().Get("list")
-	listID := MyListKey
-	switch listInput {
-	case OutFlag:
-		listID = OutListKey
-	case InFlag:
-		listID = InListKey
-	}
-
-	issues, err := p.listManager.GetIssueList(userID, listID)
+	allListIssue, err := p.listManager.GetAllList(userID)
 	if err != nil {
 		msg := "Unable to get issues for user"
 		p.API.LogError(msg, "err", err.Error())
 		p.handleErrorWithCode(w, http.StatusInternalServerError, msg, err)
 		return
 	}
 
-	if len(issues) > 0 && r.URL.Query().Get("reminder") == "true" && p.getReminderPreference(userID) {
+	if allListIssue != nil && len(allListIssue.My) > 0 && r.URL.Query().Get("reminder") == "true" && p.getReminderPreference(userID) {
```

Comment:
Interesting. I think we are changing the logic here, but I think we are actually fixing a bug. LGTM :+1: